tests/post_training/test_quantize_conformance.py

```diff
@@ -0,0 +1,334 @@
+        references[i] = target
+
+    infer_queue.wait_all()
+    predictions = np.concatenate(predictions, axis=0)
+    references = np.concatenate(references, axis=0)
+
+    return accuracy_score(predictions, references)
+
+
+def benchmark_torch_model(model, dataloader, model_name, output_path):
+    data_sample, _ = next(iter(dataloader))
+    # Dump model
+    onnx_path = Path(output_path) / (model_name + ".onnx")
+    export_to_onnx(model, onnx_path, data_sample)
+    ov_path = Path(output_path) / (model_name + ".xml")
+    export_to_ir(onnx_path, output_path, model_name)
+
+    # Benchmark performance
+    performance = benchmark_performance(ov_path, model_name)
+    # Validate accuracy
+    accuracy = validate_accuracy(ov_path, dataloader)
+    return performance, accuracy
+
+
+def benchmark_onnx_model(model, dataloader, model_name, output_path):
+    # Dump model
+    onnx_path = Path(output_path) / (model_name + ".onnx")
+    onnx.save(model, onnx_path)
+    ov_path = Path(output_path) / (model_name + ".xml")
+    export_to_ir(onnx_path, output_path, model_name)
+
+    # Benchmark performance
+    performance = benchmark_performance(ov_path, model_name)
+    # Validate accuracy
+    accuracy = validate_accuracy(ov_path, dataloader)
+    return performance, accuracy
+
+
+def benchmark_ov_model(model, dataloader, model_name, output_path):
+    # Dump model
+    ov_path = Path(output_path) / (model_name + ".xml")
+    ov.serialize(model, str(ov_path))
+
+    # Benchmark performance
+    performance = benchmark_performance(ov_path, model_name)
+    # Validate accuracy
+    accuracy = validate_accuracy(ov_path, dataloader)
+    return performance, accuracy
+
+
+@pytest.fixture(scope="session")
+def data(pytestconfig):
+    return pytestconfig.getoption("data")
+
+
+@pytest.fixture(scope="session")
+def output(pytestconfig):
+    return pytestconfig.getoption("output")
+
+
+@pytest.fixture(scope="session")
+def result(pytestconfig):
+    return pytestconfig.test_results
+
+
+@pytest.mark.parametrize("model_args", get_validation_scope())
+def test_ptq_timm(data, output, result, model_args): # pylint: disable=W0703
+    torch.multiprocessing.set_sharing_strategy(
+        "file_system"
+    )  # W/A to avoid RuntimeError
+
+    model_name = model_args["name"]
+    model_quantization_params = model_args["quantization_params"]
+    try:
+        output_folder = Path(output)
+        output_folder.mkdir(parents=True, exist_ok=True)
+
+        model = create_timm_model(model_name)
+        model.eval().cpu()
+        transform = get_model_transform(model)
+
+        batch_one_dataloader = get_torch_dataloader(data, transform, batch_size=1)
+        # benchmark original models (once)
+        orig_perf, orig_acc = benchmark_torch_model(
+            model, batch_one_dataloader, model_name, output_folder
+        )
+
+        val_dataloader = get_torch_dataloader(data, transform, batch_size=128)
```

Comment:
What's the significance of the batch size 128? Should we make this a parameter of the pytest run? Should we align it across the frameworks in this test?

Reply:
It's some reasonable default.